mgplvm/likelihoods.py

```diff
@@ -48,17 +48,12 @@ class Gaussian(Likelihood):
 
     def __init__(self,
                  n: int,
-                 variance: Optional[Tensor] = None,
+                 sigma: Optional[Tensor] = None,
                  n_gh_locs=n_gh_locs,
                  learn_sigma=True):
         super().__init__(n, n_gh_locs)
-        sigma = 1 * torch.ones(n,) if variance is None else torch.sqrt(
-            torch.tensor(variance, dtype=torch.get_default_dtype()))
-
-        if learn_sigma:
-            self.sigma = nn.Parameter(data=sigma, requires_grad=True)
-        else:
-            self.sigma = nn.Parameter(data=sigma, requires_grad=False)
+        sigma = 1 * torch.ones(n,) if sigma is None else sigma
+        self.sigma = nn.Parameter(data=sigma, requires_grad=learn_sigma)
```

Comment:
Here we should probably follow the convention we started in the kernels (also discussed in #27) and name the raw parameter `self._sigma`?  We can leave this for a future PR, where we consistently go through the entire codebase and make these changes.

Reply:
yep I agree - I just went for minimal changes here since it wasn't the purpose of this PR